src/index.js

```diff
@@ -35,7 +35,7 @@ export default class SecureLS {
       constants.EncrytionTypes.BASE64;
     this.config.encryptionSecret = config.encryptionSecret;
 
-    this.ls = localStorage;
+    this.ls = config.storageType || localStorage;
```

Comment:
Also, please update `this.ls` with `this.storage` since ls denotes localStorage and since you have added the support for sessionStorage as well, we should update the naming. Doing this will break a lot of test cases, so please find and replace all the instances of `lib.ls`  in tests folder and update it with `lib.storage`.

Reply:
I actually thought about doing that at first, but hesitated because of all the test cases. I will go ahead and change it.